packages/flutter/lib/src/material/input_decorator.dart

```diff
@@ -2901,6 +2902,13 @@ class InputDecoration {
   ///  * [helperMaxLines], the equivalent but for the [helperText].
   final int? errorMaxLines;
 
+  /// {@template flutter.material.inputDecoration.floatingLabelGap}
+  /// Defines the vertical gap between the inline elements and the floating label.
+  ///
+  /// Default is 4.0
```

Comment:
Docs should be grammatical correct sentences, so this one should end with a `.`.

Reply:
done